### PR TITLE
[Windows] Add pane to installation to allow entering of API key.

### PR DESCRIPTION
**Pull request by derekwbrown**
Adds additional panes and formatting changes.

Moves visible strings to localization file, to enable future
localization of the install

### CODE REVIEW

**Review by olivielpeau — APPROVED** [review]
👍 

I've given the MSI a try and it looks good! This doesn't actually write the api key to the `datadog.yaml` file yet correct?

**Comment by derekwbrown** [comment]
@olivielpeau yes, it does.  It writes to the registry, which gets converted to yaml on the first run (just like before).

File: omnibus/resources/agent/msi/localization-en-us.wxl.erb [hunk 1]
```diff
@@ -1,9 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <WixLocalization Culture="en-us" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="LANG">1033</String>
-  <String Id="ProductName"><%= friendly_name %></String>
-  <String Id="ManufacturerName"><%= maintainer %></String>
+  <String Id="ProductName" Overridable="no" Localizable="no">Datadog Agent</String>
+  <String Id="ManufacturerName">Overridable="no" Localizable="no">Datadog, Inc.</String>
```
Comment (olivielpeau) [inline omnibus/resources/agent/msi/localization-en-us.wxl.erb]: typo here: `Id="ManufacturerName">` -> `Id="ManufacturerName"`

**Review by olivielpeau — DISMISSED** [review]
👍  Thanks for confirming, I was able to confirm it works on a VM.

There's one typo left, once it's fixed feel free to merge!

**Comment by olivielpeau** [comment]
Code LGTM!

One last question on the behavior of the pane: does it also pop up during an agent upgrade if `datadog.yaml` already contains an api key? (and if so do we expect the user to re-enter their existing API key, or can they leave the field blank?)

**Comment by derekwbrown** [comment]
@olivielpeau I thought about your last question and it now skips the APIKEY pane altogether on upgrade.

**Review by olivielpeau — APPROVED** [review]
Great! Let's merge this for `beta.6`

**Pull request merged**